### PR TITLE
Add Managed Governance Maker Onboarding functions

### PR DESCRIPTION
**Pull request by swapnilkmahajanMS**
Add function 
1. SetManagedEnvironmentMakerOnboardingMarkdownContents : for setting maker onboarding contents
2. SetManagedEnvironmentMakerOnboardingLearnMoreUrl : for maker onboarding learn more url

### CODE REVIEW

**Comment by swapnilkmahajanMS** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Review by mihaelablendea — APPROVED** [review]

**Review by mihaelablendea — APPROVED** [review]

File: powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1 [hunk 1]
```diff
@@ -1665,6 +1665,127 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
...
+     Sets markedown contents for maker onboarding for the specified Managed environment.
+     .DESCRIPTION
+     The SetManagedEnvironmentMakerOnboardingMarkdownContents cmdlet sets markedown contents for maker onboarding for the specified Managed environment.
+     Use Get-Help SetManagedEnvironmentMakerOnboardingMarkdownContents -Examples for more details.
+     .PARAMETER EnvironmentId
+     The id (usually a GUID) of the environment.
+     .PARAMETER Markdown
+     The maker contents Markdown.
+     .EXAMPLE
+     SetManagedEnvironmentMakerOnboardingMarkdownContents -EnvironmentId 8d996ece-8558-4c4e-b459-a51b3beafdb4 -Markdown "## Welcome to Power Apps
```
Comment (kavehkazms) [inline powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1:1681]: Hmm you should try to test this locally to make sure you can set a multiline string parameter like this.
https://stackoverflow.com/questions/31793449/how-to-assign-multiple-lines-string-in-powershell-console

File: powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1 [hunk 2]
```diff
@@ -1665,6 +1665,127 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
...
+
+    $governanceConfiguration = $environment.Internal.properties.governanceConfiguration
+    $governanceConfiguration = CoalesceGovernanceConfiguration -GovernanceConfiguration $governanceConfiguration
+    if ($governanceConfiguration.protectionLevel -ne "Standard")
+    {
+        Write-Host "The specified environment is not managed."
+        return
+    }
+    
+    $makerOnboardingChangeTimestamp = (Get-Date).ToUniversalTime().ToString("ddd, dd MMM yyyy HH:mm:ss 'GMT'")
```
[the remark this answers is not in the document]
Comment (swapnilkmahajanMS) [reply powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1:1773]: I used the timestamp that came from PPAC for the existing environment.

File: powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1 [hunk 3]
```diff
@@ -1665,6 +1665,127 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
     Write-Host "Set solution checker enforcement for the specified environment."    
 }
 
+function SetManagedEnvironmentMakerOnboardingMarkdownContents
```
[the remark this answers is not in the document]
Comment (swapnilkmahajanMS) [reply powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1]: done

**Review by kavehkazms — APPROVED** [review]

**Review by JesseParsons — APPROVED** [review]

File: powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1 [hunk 4]
```diff
@@ -1665,6 +1665,126 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
     Write-Host "Set solution checker enforcement for the specified environment."    
 }
 
+function SetManagedEnvironmentMakerOnboardingMarkdownContent
+{
+    <#
+     .SYNOPSIS
+     Sets markedown content for maker onboarding for the specified Managed environment.
```
Comment (JesseParsons) [inline powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1:1672]:
```suggestion
     Sets markdown content for maker onboarding for the specified Managed environment.
```

File: powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1 [hunk 5]
```diff
@@ -1665,6 +1665,126 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
...
+
+    $response = Set-AdminPowerAppEnvironmentGovernanceConfiguration -EnvironmentName $EnvironmentId -UpdatedGovernanceConfiguration $GovernanceConfiguration
+    if ($response.Code -ne 202)
+    {
+        Write-Host "Failed to set markedown content for maker onboarding for the specified environment."
+        Write-Host $response.Internal.Message
+        return
+    }
+    
+    Write-Host "Set markedown content for maker onboarding for the specified environment."
```
Comment (JesseParsons) [inline powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1:1727]: here too, spelling

**Review by JesseParsons — CHANGES_REQUESTED** [review]

**Review by JesseParsons — APPROVED** [review]

**Pull request merged**